src/app/components/cells/text/SelectableShortText.jsx

```diff
@@ -31,33 +39,64 @@ const enhance = compose(
   needsAPIData
 );
 
+function useOutsideAlerter(callback, ref) {
+  useEffect(() => {
+    function handleClickOutside(event) {
+      if (ref.current && !ref.current.contains(event.target)) {
+        callback();
+      }
+    }
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [ref]);
```

Comment:
```suggestion
  }, [callback, ref]);
```

[major] Hier sollte ebenfalls eine Änderung des Callbacks berücksichtigt werden; das kann sonst zu schwer auffindbaren Bugs führen wenn immer wieder veraltete Callbacks gerufen werden.